modules/alloy/src/main/ts/ephox/alloy/api/testhelpers/TestHelpers.ts

```diff
@@ -1,8 +1,6 @@
 // Test code - should eventually move to a separate project
 import * as GuiSetup from './GuiSetup';
-import { TestStore } from './TestStore';
 
 export {
-  GuiSetup,
-  TestStore
```

Comment:
If you're removing this you need to log a Jira to upgrade all the premium plugins using it as well.

- [ ] Ensure followup Jira exists to upgrade anything else that uses `TestStore`.

Reply:
Logged a ticket TINY-9166 to follow up and upgrading the premium plugins that uses TestStore.

Reply:
This Jira has no details... please fill it out and ensure it has the issue and what needs to be done. It also shouldn't be assigned to the Editor Team as it could be done by anyone.

- [ ] Fix Jira

Reply:
I have added more details and DoD to the ticket TINY-9166, hope it makes more sense. I have also removed the assigned team.